### PR TITLE
chore(website): upgrade deps

### PR DESCRIPTION
**Pull request by thantos**

### CODE REVIEW

**Comment by thantos** [comment]
Core library fails with typescript 4.8, rolling back to 4.7. Seems to work locally.

**Comment by sam-goodwin** [comment]
> Core library fails with typescript 4.8, rolling back to 4.7. Seems to work locally.

Why does it fail? I'd hate to be locked into an older TS version because of website. Does it affect functionless or only the website's config?

**Comment by thantos** [comment]
> > Core library fails with typescript 4.8, rolling back to 4.7. Seems to work locally.
> 
> Why does it fail? I'd hate to be locked into an older TS version because of website. Does it affect functionless or only the website's config?

No, our core library fails with TS 4.8. We'll need to upgrade to 4.8 and resolve the errors. Website was just the first thing I tried to upgrade to 4.8. Will try with core today.

**Comment by thantos** [comment]
image

**Comment by sam-goodwin** [comment]
> image

Oh, probably the type assertion functions modifying `this` type. Maybe those are way overkill and are slowing down compilation for very little value.

**Comment by thantos** [comment]
> Oh, probably the type assertion functions modifying this type. Maybe those are way overkill and are slowing down compilation for very little value.

Not sure about the value, but definitely one of the main hotspots.

**Comment by thantos** [comment]
Removing the `!` fixed it, ha

File: website/docs/concepts/terminology.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ An Interpreter is a process that runs during [Synthesis Time](#synthesis-time) t
 
 Examples include:
 
-- [Amazon States Language](../api/classes/ASL.md)
+- [Amazon States Language](../api/classes/ASL-1.md)
```
Comment (sam-goodwin) [inline website/docs/concepts/terminology.md:78]: Why did this change

**Review by sam-goodwin — APPROVED** [review]

File: website/docs/concepts/terminology.md (hunk 1 above)
Comment (thantos) [reply website/docs/concepts/terminology.md:78]: I am not sure, version of the typedoc was upgraded, something must have changed.

**Pull request merged**